swift/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+Change Log
```

Comment:
Can you link to this changelog from the root changelog?

Reply:
Done